src/common/BaseAnnotator.ts

```diff
@@ -123,6 +132,10 @@ export default class BaseAnnotator extends EventEmitter {
         this.store.dispatch(store.setIsInitialized());
     }
 
+    public isFeatureEnabled(feature = ''): boolean {
+        return getProp(this.features, feature, false);
```

Comment:
Does something like `return this.features?.[feature] || false` work here?

Reply:
Perhaps, but I figure `getProp` could be more flexible for nesting of features?